app/assets/stylesheets/content/_ui.md

```diff
@@ -0,0 +1,21 @@
+# User Interface
```

Comment:
This is the former `_headings.md`. I have no idea where to put the Headings for the rest of the application otherwise.

@ulferts thoughts?